packages/utilities/psammead-locales/README.md

```diff
@@ -6,7 +6,7 @@ This package provides a collection of locale configs, used in BBC World Service
 
 `/moment/ig` - A [moment.js](https://momentjs.com/) locale for Igbo. This can be used to localise timestamps generated using moment.js.
 `/moment/pcm` - A [moment.js](https://momentjs.com/) locale for Pidgin. This can be used to localise timestamps generated using moment.js.
-
+`/moment/yo` - A [moment.js](https://momentjs.com/) locale override for Yoruba. This replaces some translations as requested for BBC News World Service.
```

Comment:
Can we put something about how `'moment/locale/yo';` needs to be available too? 

Or could we maybe import `'moment/locale/yo';` into `packages/utilities/psammead-locales/moment/yo.js` so you dont need to import 2 things?

Reply:
E.g. we currently do
```
import 'moment/locale/yo';
```
so we would need to do
```
import 'moment/locale/yo';
import '@bbc/psammead-locales/moment/yo';
```

Reply:
I believe `updateLocale` is doing it implicitly already? The story has the locale without explicitly importing the locale

Reply:
But I agree it would be better to do it explicitly in the `packages/utilities/psammead-locales/moment/yo.js` :+1: